pytest.ini

```diff
@@ -23,6 +23,7 @@ markers =
     ratsignal_parse: marks a test as a ratsignal parser test
     autocorrect: tests for the autocorrect util
     offline: offline-aware tests
+    mechaclient: marks a test as a mechaclient test.
```

Comment:
```suggestion
    mechaclient: mechaclient tests
```

Reply:
Changed to 'mechaclient tests' in 96dac7b.